<!DOCTYPE HTML>
<!--
	Minimaxing by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>ANiroula</title>
		<meta name="keywords" content="Clonal hematopoiesis, CHIP, Abhishek Niroula, Niroula lab">
		<meta name="description" content="ANiroula">
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body>
		<div id="page-wrapper">

			<!-- Header -->
				<div id="header-wrapper">
					<div class="container">
						<div class="row">
							<div class="col-12">

								<header id="header">
									<h1 style="font-size:30px;"><a href="index.html" id="logo">Niroula lab</a></h1>
									<nav id="nav">
										<a href="index.html">Home</a>
										<a href="research.html">Research</a>
										<a href="publications.html">Publications</a>
										<a href="team.html">Members</a>
									</nav>
								</header>

							</div>
						</div>
					</div>
				</div>

			<!-- Main -->
				<div id="main">
					<div class="container">
						<div class="row main-row">
							<div class="col-1 col-12-medium">
							</div>
							<div class="col-10 col-12-medium imp-medium">

								<section>
									<h2>Publications</h2>
									<p style="font-size:22px;"> For a complete list of publications, see <a href="https://scholar.google.com/citations?user=XtbiqvYAAAAJ&hl=en">here</a>.
										
										<!--<li><p><b>.</b> Niroula A, <i>et al. JOURNAL</i> (20). doi: <a href="doi.org/">doi.org/x</a></p></li>-->

										<ol type="1">
										
										<li><p><b>Clonal haematopoiesis of indeterminate potential predicts incident cardiac arrhythmias.</b> Schuermans A, Vlasschaert C, Nauffal V, et al. Eur Heart J (2023). doi: <a href="https://doi.org/10.1093/eurheartj/ehad670">doi.org/10.1093/eurheartj/ehad670</a></p></li>

										<li><p><b>SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation.</b> Chen CW, Zhang L, Dutta R, et al. Cell Stem Cell (2023). doi: <a href="https://doi.org/10.1016/j.stem.2023.09.011">doi.org/10.1016/j.stem.2023.09.011</a></p></li>

										<li><p><b>Functional and molecular profiling of hematopoietic stem cells during regeneration.</b> Rydström A, Grahn THM, Niroula A, et al. Exp Hematol (2023). doi: <a href="https://doi.org/10.1016/j.exphem.2023.08.010">doi.org/10.1016/j.exphem.2023.08.010</a></p></li>

										<li><p><b>Genetic modification of inflammation-and clonal hematopoiesis–associated cardiovascular risk.</b> Yu Z, Fidler TP, Ruan Y, et al. J Clin Invest (2023). doi: <a href="https://doi.org/10.1172/JCI168597">doi.org/10.1172/JCI168597</a></p></li>

										<li><p><b>Birth Weight Is Associated With Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Outcomes in Adulthood.</b> Schuermans A, Nakao T, Ruan Y, et al. J Am Heart Assoc (2023). doi: <a href="https://doi.org/10.1161/JAHA.123.030220">doi.org/10.1161/JAHA.123.030220</a></p></li>

										<li><p><b>Obesity-induced inflammation exacerbates clonal hematopoiesis.</b> Pasupuleti SK, Ramdas B, Burns SS, et al. J Clin Invest (2023). doi: <a href="https://doi.org/10.1172/JCI163968">doi.org/10.1172/JCI163968</a></p></li>

										<li><p><b>Clonal hematopoiesis of indeterminate potential predicts adverse outcomes in patients with atherosclerotic cardiovascular disease.</b> Gumuser ED, Schuermans A, Cho SMJ, et al. J Am Coll Cardiol (2023). doi: <a href="https://doi.org/10.1016/j.jacc.2023.03.401">doi.org/10.1016/j.jacc.2023.03.401</a></p></li>

										<li><p><b>A practical approach to curate clonal hematopoiesis of indeterminate potential in human genetic data sets.</b> Vlasschaert C, Mack T, Heimlich JB, et al. Blood (2023). doi: <a href="https://doi.org/10.1182/blood.2022018825">doi.org/10.1182/blood.2022018825</a></p></li>

										<li><p><b>Prediction of risk for myeloid malignancy in clonal hematopoiesis.</b> Weeks LD, Niroula A, Neuberg D, et al. NEJM Evid (2023). doi: <a href="https://doi.org/10.1056/evidoa2200310">doi.org/10.1056/evidoa2200310</a></p></li>

										<li><p><b>Clonal haematopoiesis and risk of chronic liver disease.</b> Wong WJ, Emdin C, Bick AG, et al. Nature (2023). doi: <a href="https://doi.org/10.1038/s41586-023-05857-4">doi.org/10.1038/s41586-023-05857-4</a></p></li>

										<li><p><b>HLA Class I Genotype Is Associated with Relapse Risk after Allogeneic Stem Cell Transplantation for NPM1-Mutated Acute Myeloid Leukemia.</b> Narayan R, Niroula A, Wang T, et al. Transplant Cell Ther (2023). doi: <a href="https://doi.org/10.1016/j.jtct.2023.03.027">10.1016/j.jtct.2023.03.027</a></p></li>
										
										<li><p><b>Clonal hematopoiesis and risk of incident lung cancer.</b> Tian R, Wiley B, Liu J, et al. J Clin Oncol (2023). doi: <a href="https://doi.org/10.1200/JCO.22.00857">doi.org/10.1200/JCO.22.00857</a></p></li>

										<li><p><b>TP53-mediated clonal hematopoiesis confers increased risk for incident atherosclerotic disease.</b> Zekavat SM, Viana-Huete V, Matesanz N, et al. Nat Cardiovasc Res (2023). doi: <a href="https://10.1038/s44161-022-00206-6">doi.org/10.1038/s44161-022-00206-6</a></p></li>

										<li><p><b>TET2-mutant clonal hematopoiesis and risk of gout.</b> Agrawal M, Niroula A, Cunin P, et al. Blood (2022). doi: <a href="https://doi.org/10.1182/blood.2022015384">doi.org/10.1182/blood.2022015384.</a></p></li>

										<li><p><b>Genetics of smoking and risk of clonal hematopoiesis.</b> Levin MG, Nakao T, Zekavat SM, et al. Sci Rep (2022). doi: <a href="https://doi.org/10.1038/s41598-022-09604-z">doi.org/10.1038/s41598-022-09604-z.</a></p></li>

										<li><p><b>Clonal Hematopoiesis and CKD Progression.</b> Niroula A, Belizaire R. J Am Soc Nephrol (2022). doi: <a href="https://doi.org/10.1681/ASN.2022030262">doi.org/10.1681/ASN.2022030262.</a></p></li>

										<li><p><b>Mendelian randomization supports bidirectional causality between telomere length and clonal hematopoiesis of indeterminate potential.</b> Nakao T, Bick AG, Taub MA, et al. Sci Adv (2022). doi: <a href="https://doi.org/10.1126/sciadv.abl6579">doi.org/10.1126/sciadv.abl6579.</a></p></li>

										<li><p><b>Genome-wide association study on 13,167 individuals identifies regulators of hematopoietic stem and progenitor cell levels in human blood.</b> Lopez de Lapuente Portilla A, Ekdahl L, Cafaro C, et al. Blood (2022). doi: <a href="https://doi.org/10.1182/blood.2021013220">doi.org/10.1182/blood.2021013220.</a></p></li>

										<li><p><b>Functional dissection of inherited non-coding variation influencing multiple myeloma risk.</b> Ajore R, Niroula A, Pertesi M, et al. Nat Comms (2022). doi: <a href="https://doi.org/10.1038/s41467-021-27666-x">doi.org/10.1038/s41467-021-27666-x.</a></p></li>

										<li><p><b>Association of clonal hematopoiesis with chronic obstructive pulmonary disease.</b> Miller PG, Qiao D, Rojas-Quintero J, et al. Blood (2022). doi: <a href="https://doi.org/10.1182/blood.2021013531">doi.org/10.1182/blood.2021013531.</a></p></li>

										<li><p><b>Clonal hematopoiesis is associated with higher risk of stroke.</b> Bhattacharya R, Zekavat SM, Haessler J, et al. Stroke (2022). doi: <a href="https://doi.org/10.1161/STROKEAHA.121.037388">doi.org/10.1161/STROKEAHA.121.037388.</a></p></li>

										<li><p><b><i>Dnmt3a</i>-mutated clonal hematopoiesis promotes osteoporosis.</b> Kim PG, Niroula A, Shkolnik V, et al. J Exp Med (2021). doi: <a href="https://doi.org/10.1084/jem.20211872">doi.org/10.1084/jem.20211872.</a></p></li>

										<li><p><b>Distinction of lymphoid and myeloid clonal hematopoiesis.</b> Niroula A, Sekar A, Murakami MA, et al. Nat Med (2021). doi: <a href="https://doi.org/10.1038/s41591-021-01521-4">doi.org/10.1038/s41591-021-01521-4</a></p></li>

										<li><p><b><i>ZBTB33</i> is mutated in clonal hematopoiesis and myelodysplastic syndromes and impacts RNA splicing.</b> Beauchamp EM, Leventhal M, Bernard E, et al. Blood Cancer Discov (2021). doi: <a href="https://doi.org/10.1158/2643-3230.BCD-20-0224">doi.org/10.1158/2643-3230.BCD-20-0224.</a></p></li>

										<li><p><b>Association of clonal hematopoiesis with incident heart failure.</b> Yu B, Roberrts MB, Raffield LM, et al. J Am Coll Cardiol (2021). doi: <a href="https://doi.org/10.1016/j.jacc.2021.04.085">doi.org/10.1016/j.jacc.2021.04.085.</a></p></li>

										<li><p><b>Prognostic implications of troponin T variations in inherited cardiomyopathies using systems biology.</b> Shakur R, Ochoa JP, Robinson AJ, et al. NPJ Genom Med (2021). doi: <a href="https://doi.org/10.1038/s41525-021-00204-w">doi.org/10.1038/s41525-021-00204-w.</a></p></li>

										<li><p><b>Association of diet quality with prevalence of clonal hematopoiesis and adverse cardiovascular events.</b> Bhattacharya R, Zekavat SM, Uddin MM, et al. JAMA Cardiol (2021). doi: <a href="https://doi.org/10.1001/jamacardio.2021.1678">doi.org/10.1001/jamacardio.2021.1678.</a></p></li>

										<li><p><b>Germline variants at SOHLH2 influence multiple myeloma risk.</b> Duran-Lozano L, Thorleifsson G, Lopez de Lapuente Portilla A, et al. Blood Cancer J (2021). doi: <a href="https://doi.org/10.1038/s41408-021-00468-6">doi.org/10.1038/s41408-021-00468-6.</a></p></li>

										<li><p><b>Accelerating target deconvolution for therapeutic antibody candidates using highly parallelized genome editing.</b> Mattssson J, Ekdahl L, unghus F, et al. Nat Commun (2021). doi: <a href="https://doi.org/10.1038/s41467-021-21518-4">doi.org/10.1038/s41467-021-21518-4.</a></p></li>

										<li><p><b>Premature menopause, clonal hematopoiesis, and coronary artery disease in postmenopausal women.</b> Honigberg MC, Zekavat SM, Niroula A, et al. Circulation (2021). doi: <a href="https://doi.org/10.1161/CIRCULATIONAHA.120.051775">doi.org/10.1161/CIRCULATIONAHA.120.051775.</a></p></li>

										<li><p><b>Inherited causes of clonal haematopoiesis in 97,691 whole genomes.</b> Bick AG, Weinstock JS, Nandakumar SK. Nature (2020). doi: <a href="https://doi.org/10.1038/s41586-020-2819-2">doi.org/10.1038/s41586-020-2819-2.</a></p></li>

										<li><p><b>S100A6 is a critical regulator of hematopoietic stem cells.</b> Grahn THM, Niroula A, Végvári Á, et al. Leukemia (2020). doi: <a href="https://doi.org/10.1038/s41375-020-0901-2">doi.org/10.1038/s41375-020-0901-2.</a></p></li>

										<li><p><b>Identification of germline variants in adults with hemophagocytic lymphohistiocytosis.</b> Miller PG, Niroula A, Ceremsak JJ, et al. Blood Adv (2020). doi: <a href="https://doi.org/10.1182/bloodadvances.2019001272">doi.org/10.1182/bloodadvances.2019001272.</a></p></li>

										<li><p><b>ProTstab - predictor for cellular protein stability.</b> Yang Y, Ding X, Zhu G, et al. BMC Genomics (2019). doi: <a href="https://doi.org/10.1186/s12864-019-6138-7">doi.org/10.1186/s12864-019-6138-7.</a></p></li>

										<li><p><b>MPRAscore: robust and non-parametric analysis of massively parallel reporter assays.</b> Ajore R, Niroula A, Nilsson B. Bioinformatics (2019). doi: <a href="https://doi.org/10.1093/bioinformatics/btz591">doi.org/10.1093/bioinformatics/btz591.</a></p></li>

										<li><p><b>How good are pathogenicity predictors in detecting benign variants?.</b> Niroula A, Vihinen M. PLoS Comput Biol (2019). doi: <a href="https://doi.org/10.1371/journal.pcbi.1006481">doi.org/10.1371/journal.pcbi.1006481.</a></p></li>

										<li><p><b>The multiple myeloma risk allele at 5q15 lowers ELL2 expression and increases ribosomal gene expression.</b> Ali M, Ajore R, Wihlborg AK, et al. Nat Commun (2018). doi: <a href="https://doi.org/10.1038/s41467-018-04082-2">doi.org/10.1038/s41467-018-04082-2.</a></p></li>

										<li><p><b>PON-tstab: Protein variant stability predictor. Importance of training data quality.</b> Yang Y, Urolagin S, Niroula A, et al. Int J Mol Sci (2018). doi: <a href="https://doi.org/10.3390/ijms19041009">doi.org/10.3390/ijms19041009.</a></p></li>

										<li><p><b>Direct evidence for a polygenic etiology in familial multiple myeloma.</b> Halvarsson BM, Wihlborg AK, Ali M, et al. Blood Adv (2017). doi: <a href="https://doi.org/10.1182/bloodadvances.2016003111">doi.org/10.1182/bloodadvances.2016003111.</a></p></li>

										<li><p><b>Working towards precision medicine: predicting phenotypes from exomes in the Critical Assessment of Genome Interpretation (CAGI) challenges.</b> Daneshjou R, Wang Y, Bromberg, Y, et al. Hum Mutat (2017). doi: <a href="https://doi.org/10.1002/humu.23280">doi.org/10.1002/humu.23280.</a></p></li>

										<li><p><b>Performance of in silico tools for the evaluation of p16INK4a (CDKN2A) variants in CAGI.</b> Carraro M, Minervini G, Giollo M, et al. Hum Mutat (2017). doi: <a href="https://doi.org/10.1002/humu.23235">doi.org/10.1002/humu.23235.</a></p></li>

										<li><p><b>PON-P and PON-P2 predictor performance in CAGI challenges: Lessons learned.</b> Niroula A, Vihinen M. Hum Mutat (2017). doi: <a href="https://doi.org/10.1002/humu.23199">doi.org/10.1002/humu.23199.</a></p></li>

										<li><p><b>Predicting severity of disease‐causing variants.</b> Niroula A, Vihinen M. Hum Mutat (2017). doi: <a href="https://doi.org/10.1002/humu.23173">doi.org/10.1002/humu.23173.</a></p></li>

										<li><p><b>PON-Sol: prediction of effects of amino acid substitutions on protein solubility.</b> Yang Y, Niroula A, Shen B, et al. Bioinformatics (2016). doi: <a href="https://doi.org/10.1093/bioinformatics/btw066">doi.org/10.1093/bioinformatics/btw066.</a></p></li>

										<li><p><b>Variation interpretation predictors: principles, types, performance, and choice.</b> Niroula A, Vihinen M. Hum Mutat (2016). doi: <a href="https://doi.org/10.1002/humu.22987">doi.org/10.1002/humu.22987.</a></p></li>

										<li><p><b>PON-mt-tRNA: a multifactorial probability-based method for classification of mitochondrial tRNA variations.</b> Niroula A, Vihinen M. Nucleic Acids Res (2016). doi: <a href="https://doi.org/10.1093/nar/gkw046">doi.org/10.1093/nar/gkw046.</a></p></li>

										<li><p><b>Classification of amino acid substitutions in mismatch repair proteins using PON‐MMR2.</b> Niroula A, Vihinen M. Hum Mutat (2015). doi: <a href="https://doi.org/10.1002/humu.22900">doi.org/10.1002/humu.22900.</a></p></li>

										<li><p><b>Harmful somatic amino acid substitutions affect key pathways in cancers.</b> Niroula A, Vihinen M. BMC Medical Genomics (2015). doi: <a href="https://doi.org/10.1186/s12920-015-0125-x">doi.org/10.1186/s12920-015-0125-x.</a></p></li>

										<li><p><b>PON-P2: prediction method for fast and reliable identification of harmful variants.</b> Niroula A, Urolagin S, Vihinen M. PloS one (2015). doi: <a href="https://doi.org/10.1371/journal.pone.0117380">doi.org/10.1371/journal.pone.0117380.</a></a> </p></li>

									</ol>

								</section>
							</div>
							
							
						</div>
					</div>
				</div>

			<!-- Footer -->
				<div id="footer-wrapper">
					<div class="container">
						<div class="row">
							<div class="col-12">

								<div id="copyright">
									&copy; CGB-GU. All rights reserved. | Design: <a href="http://html5up.net">HTML5 UP</a>
								</div>

							</div>
						</div>
					</div>
				</div>
		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>