document.write(`

   <li>ProTstab - predictor for cellular protein stability. Yang Y, Ding X, Zhu G, et al. BMC Genomics (2019). <a href="https://doi.org/10.1186/s12864-019-6138-7">doi.org/10.1186/s12864-019-6138-7.</a></li><br>
   <li>MPRAscore: robust and non-parametric analysis of massively parallel reporter assays. Ajore R, Niroula A, Nilsson B. Bioinformatics (2019). <a href="https://doi.org/10.1093/bioinformatics/btz591">doi.org/10.1093/bioinformatics/btz591.</a></li><br>
   <li>How good are pathogenicity predictors in detecting benign variants?. Niroula A, Vihinen M. PLoS Comput Biol (2019). <a href="https://doi.org/10.1371/journal.pcbi.1006481">doi.org/10.1371/journal.pcbi.1006481.</a></li><br>

   <li>The multiple myeloma risk allele at 5q15 lowers ELL2 expression and increases ribosomal gene expression. Ali M, Ajore R, Wihlborg AK, et al. Nat Commun (2018). <a href="https://doi.org/10.1038/s41467-018-04082-2">doi.org/10.1038/s41467-018-04082-2.</a></li><br>
   <li>PON-tstab: Protein variant stability predictor. Importance of training data quality. Yang Y, Urolagin S, Niroula A, et al. Int J Mol Sci (2018). <a href="https://doi.org/10.3390/ijms19041009">doi.org/10.3390/ijms19041009.</a></li><br>

   <li>Direct evidence for a polygenic etiology in familial multiple myeloma. Halvarsson BM, Wihlborg AK, Ali M, et al. Blood Adv (2017). <a href="https://doi.org/10.1182/bloodadvances.2016003111">doi.org/10.1182/bloodadvances.2016003111.</a></li><br>
   <li>Working towards precision medicine: predicting phenotypes from exomes in the Critical Assessment of Genome Interpretation (CAGI) challenges. Daneshjou R, Wang Y, Bromberg, Y, et al. Hum Mutat (2017). <a href="https://doi.org/10.1002/humu.23280">doi.org/10.1002/humu.23280.</a></li><br>
   <li>Performance of in silico tools for the evaluation of p16INK4a (CDKN2A) variants in CAGI. Carraro M, Minervini G, Giollo M, et al. Hum Mutat (2017). <a href="https://doi.org/10.1002/humu.23235">doi.org/10.1002/humu.23235.</a></li><br>
   <li>PON-P and PON-P2 predictor performance in CAGI challenges: Lessons learned. Niroula A, Vihinen M. Hum Mutat (2017). <a href="https://doi.org/10.1002/humu.23199">doi.org/10.1002/humu.23199.</a></li><br>
   <li>Predicting severity of disease‐causing variants. Niroula A, Vihinen M. Hum Mutat (2017). <a href="https://doi.org/10.1002/humu.23173">doi.org/10.1002/humu.23173.</a></li><br>

   <li>PON-Sol: prediction of effects of amino acid substitutions on protein solubility. Yang Y, Niroula A, Shen B, et al. Bioinformatics (2016). <a href="https://doi.org/10.1093/bioinformatics/btw066">doi.org/10.1093/bioinformatics/btw066.</a></li><br>
   <li>Variation interpretation predictors: principles, types, performance, and choice. Niroula A, Vihinen M. Hum Mutat (2016). <a href="https://doi.org/10.1002/humu.22987">doi.org/10.1002/humu.22987.</a></li><br>
   <li>PON-mt-tRNA: a multifactorial probability-based method for classification of mitochondrial tRNA variations. Niroula A, Vihinen M. Nucleic Acids Res (2016). <a href="https://doi.org/10.1093/nar/gkw046">doi.org/10.1093/nar/gkw046.</a></li><br>


   <li>Classification of amino acid substitutions in mismatch repair proteins using PON‐MMR2. Niroula A, Vihinen M. Hum Mutat (2015). <a href="https://doi.org/10.1002/humu.22900">doi.org/10.1002/humu.22900.</a></li><br>
   <li>Harmful somatic amino acid substitutions affect key pathways in cancers. Niroula A, Vihinen M. BMC Medical Genomics (2015). <a href="https://doi.org/10.1186/s12920-015-0125-x">doi.org/10.1186/s12920-015-0125-x.</a></li><br>
   <li>PON-P2: prediction method for fast and reliable identification of harmful variants. Niroula A, Urolagin S, Vihinen M. PloS one (2015). <a href="https://doi.org/10.1371/journal.pone.0117380">doi.org/10.1371/journal.pone.0117380.</a></li><br>

`);
